### ISSUE TITLE
Change event used when renewing your purchase

### ISSUE THREAD

**Issue by NickCuso** [unlock-protocol/unlock#5728]
**Describe the bug**
If you purchase 2 copies of the same key, some screens on etherscan consider this 2 tokens: https://etherscan.io/token/0x91002310cec24d818f42a169dd0d27ecbfa8bcd5?a=0xdb92c096bc5efa8adb48f05cd601dddb75228203#inventory

**To Reproduce**
1. purchase a key
2. purchase it again

**Expected behavior**
Per EIP-721 we should emit Transfer(from: 0) when minting.  But our use case is a bit unique.

I think we could address this by creating a new event type.  Emit Transfer only for the first purchase (when the tokenId is assigned).  Then when a token is renewed we can emit a new `Renewal` event instead.

**Comment by nfurfaro** [comment]
We could stop emitting Transfer for renewals and just emit TimestampChanged in the timeMachine function. We already fire this event, we would just need to refactor a bit to use the timeMachine function inside `purchase` when it's a renewal/top-up.

**Comment by julien51** [comment]
Let's use `ExpirationChanged` so it is more explicit than `timestamp`...

**Comment by nfurfaro** [comment]
Yeah, we could change the existing event name. It's probably not being used anywhere yet so I doubt it would break anything except a few tests.

**Comment by nfurfaro** [comment]
I just noticed this as well.

There is another location in the code which could potentially cause some confusion with the transfer event, though I don’t think we’re seeing it show up yet. In the shareKey function, I believe the Transfer event  will sometimes emit twice under certain(likely common) conditions.

When sharing with a recipient who has no key, we emit `Transfer(address(0), _to, iDTo)` to signal the new key creation.
We then proceed to emit `emit Transfer( keyOwner, _to, iDTo` 
below this to signal the transfer of time from the existing key to the new one.
I think the second event should actually be removed, as the call to timeMachine already emits the `TimestampChanged` event once for each key under most cases. 

https://github.com/unlock-protocol/unlock/blob/3bf5e79ad77321ca4256c0aad25dd38797640cb8/smart-contracts/contracts/mixins/MixinTransfer.sol#L93
